lib/apipie/generator/swagger/param_description/builder.rb

```diff
@@ -78,6 +79,14 @@ def for_default
     }
   end
 
+  def for_example
+    return {} unless @param_description.options.key?(:ex)
+
+    {
+      example: @param_description.options[:ex],
```

Comment:
```suggestion
      example: @param_description.options[:example],
```

Reply:
I agree it will be better to name it explicitly 😃. My only concern is that there is already an `example` keywords in the method description, I am afraid that using the same name will cause some confusion. 🤔

Reply:
There is an `examples

Reply:
There is an `examples` attribute in `Apipie::MethodDescription`, that is never used, if I recall correctly, in the generator. I think it ok to go with `example`